<?php

namespace Drupal\Tests\jcms_migrate\Unit\process;

use Drupal\jcms_migrate\Plugin\migrate\process\JCMSPressPackageContent;
use Drupal\Tests\migrate\Unit\process\MigrateProcessTestCase;

/**
 * Tests for the press package content process plugin.
 *
 * @coversDefaultClass \Drupal\jcms_migrate\Plugin\migrate\process\JCMSPressPackageContent
 * @group jcms_migrate
 */
class JCMSPressPackageContentTest extends MigrateProcessTestCase {

  /**
   * @test
   * @covers ::transform
   * @dataProvider breakupContentDataProvider
   * @group journal-cms-tests
   */
  public function testBreakupContent($content, $expected_result) {
    $plugin = new JCMSPressPackageContent([], 'jcms_split_collection_content', []);
    $breakup_content = $plugin->breakupContent($content);
    $this->assertEquals($expected_result, array_intersect_key($breakup_content, array_flip(array_keys($expected_result))));
  }

  public function breakupContentDataProvider() {
    return [
      [
        "<div>\n<p><strong>Scientists have revealed the brain activity in animals that helps them find food and other vital resources in unfamiliar environments where there are no cues, such as lights and sounds, to guide them.</strong>&nbsp;</p>\n</div>\n\n<div>\n<p>Animals that are&nbsp;placed&nbsp;in&nbsp;such&nbsp;environments&nbsp;display spontaneous, seemingly random&nbsp;behaviors&nbsp;when foraging.&nbsp;These behaviors have been observed in many organisms,&nbsp;although&nbsp;the brain activity&nbsp;behind them has&nbsp;remained elusive due to difficulties in&nbsp;knowing&nbsp;where to look for neural signals in large vertebrate brains.&nbsp;&nbsp;</p>\n</div>\n\n<div>\n<p>Now, in a&nbsp;study to be published in the journal&nbsp;eLife, researchers&nbsp;have used whole-brain imaging in larval&nbsp;zebrafish&nbsp;to&nbsp;discover how their brain activity translates&nbsp;into spontaneous behaviors. They found that&nbsp;the&nbsp;animals’&nbsp;behavior&nbsp;in plain surroundings&nbsp;is&nbsp;not&nbsp;random&nbsp;at all, but is&nbsp;characterized by alternating left and right&nbsp;turn “states” in the brain,&nbsp;where the animals&nbsp;are more likely to perform repeated left and right turning&nbsp;maneuvers, respectively.&nbsp;&nbsp;</p>\n</div>",
        [
          'content' => "<div>\n<p><strong>Scientists have revealed the brain activity in animals that helps them find food and other vital resources in unfamiliar environments where there are no cues, such as lights and sounds, to guide them.</strong> </p>\n</div>\n\n<div>\n<p>Animals that are placed in such environments display spontaneous, seemingly random behaviors when foraging. These behaviors have been observed in many organisms, although the brain activity behind them has remained elusive due to difficulties in knowing where to look for neural signals in large vertebrate brains. </p>\n</div>\n\n<div>\n<p>Now, in a study to be published in the journal eLife, researchers have used whole-brain imaging in larval zebrafish to discover how their brain activity translates into spontaneous behaviors. They found that the animals’ behavior in plain surroundings is not random at all, but is characterized by alternating left and right turn “states” in the brain, where the animals are more likely to perform repeated left and right turning maneuvers, respectively. </p>\n</div>",
          'relatedContent' => NULL,
        ],
      ],
      [
        "<p><strong>The wide diversity of flu in pigs across multiple continents, mostly introduced from humans, highlights the significant potential of new swine flu strains emerging, according to a study&nbsp;published in eLife.</strong></p>\n\n<p>While swine flu viruses have long been considered a risk for human pandemics, and were the source of the 2009 pandemic H1N1 virus, attention has recently turned to the transmission of flu viruses from humans to pigs.</p>\n\n<p>&#x201C;Once in pigs, flu viruses from humans continue to evolve their surface proteins, generically referred to as antigens, resulting in a tremendous diversity of novel flu viruses that can be transmitted to other pigs and also to humans,&#x201D; explains first author Nicola Lewis from the University of Cambridge. &nbsp;</p>\n\n<p>&#x201C;These flu viruses pose a serious threat to public health because they are no longer similar enough to the current human flu strains for our immune systems to recognise them and mount an effective defence. Understanding the dynamics and consequences of this two-way transmission is important for designing effective strategies to detect and respond to new strains of flu.&#x201D;</p>\n\n<p>Humans and pigs both experience regular outbreaks of influenza A viruses, most commonly from H1 and H3 subtypes. Their genetic diversity is well characterised. However, the diversity of their antigens, which shapes their pandemic potential, is poorly understood, mainly due to lack of data.</p>\n\n<p>To help improve this understanding, Lewis and her team created the largest and most geographically comprehensive dataset of antigenic variation. They amassed and characterised antigens from nearly 600 flu viruses dating back from 1930 through to 2013 and collected from multiple continents, including Europe, the US, and Asia. They included nearly 200 viruses that had never been studied before.</p>\n\n<p>Analysis of their data reveals that the amount of antigenic diversity in swine flu viruses resembles the diversity of H1 and H3 viruses seen in humans over the last 40 years, driven by the frequent introduction of human viruses to pigs. In contrast, flu from birds has rarely contributed substantially to the diversity in pigs. However, little is currently known about the antigenic relationship between flu in birds and pigs.</p>\n\n<p>&#x2028;&#x201C;Since most of the current swine flu viruses are the result of human seasonal flu virus introductions into pigs, we anticipate at least some&nbsp;cross-protective immunity in the human population, which could potentially interfere with a re-introduction of these viruses. For example, the H1N1pdm09 viruses circulating in both humans and pigs are antigenically similar and therefore&nbsp;likely induce some immunity in both hosts,&#x201D; says Lewis.</p>\n\n<p>&#x201C;However, for the H1 1C, H3 3A, and H3 3B human seasonal lineages in pigs,&nbsp;the risk of re-introduction into the human population increases with the number of people born after the circulation of the human precursor virus, and is increased by the antigenic evolution of these viruses in pigs. Earlier introduced lineages of human H1 and H3 viruses therefore pose the greatest current risk to humans, due to the low or negligible predicted levels of cross-immunity in individuals born since the 1970s.&#x201D;</p>\n\n<p>Swine flu causes symptoms such as coughing, fever, body aches, chills, and fatigue in humans. Pigs can also experience fever and coughing (barking), along with discharge from the nose or eyes, breathing difficulties, eye redness or inflammation, and going off feed &#x2013; although some display no clinical signs at all.</p>\n\n<p>Vaccination to control flu in pigs is used extensively in the US and occasionally in other regions. Control strategies vary by region, with some countries not using any vaccinations, while others produce herd-specific vaccines for individual producers. There is no formal system for matching vaccine strains with circulating strains, however, and no validated protocols for standardisation and effective vaccine use.</p>\n\n<p>&#x201C;The significant antigenic diversity that we see in our data means it is&nbsp;highly unlikely that one vaccine strain per subtype would be effective on a global scale, or even&nbsp;in a given region,&#x201D; says co-author Colin Russell, also from the University of Cambridge.</p>\n\n<p>&#x201C;Our findings therefore have important implications for developing flu vaccines for pigs. They also emphasise the need for more focused surveillance in areas with a high pig population density, such as China, and situations where humans and pigs have close contact, in order to better assess the incidence of transmission between the animals and risk of spreading to humans.&#x201D;</p>\n\n<p>##</p>\n\n<p><strong>Reference</strong></p>\n\n<p>The paper &#x2018;The global antigenic diversity of swine influenza A viruses&#x2019; can be freely accessed online at <a href=\"http://dx.doi.org/10.7554/eLife.12217\">http://dx.doi.org/10.7554/eLife.12217</a>. Contents, including text, figures, and data, are free to reuse under a CC BY 4.0 license.</p>\n\n<p><strong>Media contacts</strong></p>\n\n<p>Craig Brierley, University of Cambridge</p>\n\n<p><a href=\"mailto:craig.brierley@admin.cam.ac.uk\">craig.brierley@admin.cam.ac.uk</a></p>\n\n<p>01223 332300</p>\n\n<p>Emily Packer, eLife</p>\n\n<p><a href=\"mailto:e.packer@elifesciences.org\">e.packer@elifesciences.org</a></p>\n\n<p>01223 855373</p>\n\n<p><strong>About eLife</strong></p>\n\n<p>eLife is a unique collaboration between the funders and practitioners of research to improve the way important research is selected, presented, and shared. eLife publishes outstanding works across the life sciences and biomedicine &#x2014; from basic biological research to applied, translational, and clinical studies. All papers are selected by active scientists in the research community. Decisions and responses are agreed by the reviewers and consolidated by the Reviewing Editor into a single, clear set of instructions for authors, removing the need for laborious cycles of revision and allowing authors to publish their findings quickly. eLife is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.</p>\n",
        [
          'relatedContent' => [
            [
              'type' => 'article',
              'source' => '12217',
            ],
          ],
          'mediaContacts' => [
            [
              'name' => [
                'preferred' => 'Craig Brierley',
                'index' => 'Brierley, Craig',
              ],
              'emailAddresses' => [
                'craig.brierley@admin.cam.ac.uk',
              ],
              'phoneNumbers' => [
                '+441223332300',
              ],
              'affiliations' => [
                [
                  'name' => [
                    'University of Cambridge',
                  ],
                ],
              ],
            ],
          ],
        ],
      ],
      [
        "<p>A new and inexpensive technique for mass-producing the main ingredient in the most effective treatment for malaria, artemisinin, could help meet global demands for the drug, according to a study to be published in the journal eLife.&nbsp;</p>\n\n<p>Artemisinin is produced in low yields by a herb called <em>Artemisia annua</em> (<em>A. annua</em>), otherwise known as sweet wormwood. Researchers from the Max Planck Institute of Molecular Plant Physiology have now discovered a new way to produce artemisinic acid, the molecule from which artemisinin is derived, in high yields. Their method involves transferring its metabolic pathway &#x2013; the series of biochemical steps involved in its production &#x2013; from <em>A. annua</em> into tobacco, a high-biomass crop. &nbsp;</p>\n\n<p>&#x201C;Malaria is a devastating tropical disease that kills almost half a million people every year,&#x201D; says contributing author Ralph Bock, Director of the Department for Organelle Biology, Biotechnology and Molecular Ecophysiology.&nbsp;</p>\n\n<p>&#x201C;For the foreseeable future, artemisinin will be the most powerful weapon in the battle against malaria but, due to its extraction from low-yielding plants, it is currently too expensive to be widely accessible to patients in poorer countries. Producing artemisinic acid in a crop such as tobacco, which yields large amounts of leafy biomass, could provide a sustainable and inexpensive source of the drug, making it more readily available for those who need it most.&#x201D;&nbsp;</p>\n\n<p>The team has called this approach to producing more artemisinic acid COSTREL (&#x201C;combinatorial supertransformation of transplastomic recipient lines&#x201D;). The first step in their process was to transfer the genes of the artemisinic acid pathway&#x2019;s core set of enzymes into the chloroplast genome of tobacco plants, generating what are known as transplastomic plants.&nbsp;</p>\n\n<p>The team then used their best transplastomic tobacco plant line to introduce an additional set of genes into its nuclear genome, generating the COSTREL lines. These remaining genes encode factors that increase the synthesis, or generation, of the acid in ways that are still largely unknown.&nbsp;</p>\n\n<p>&#x201C;While the artemisinic acid pathway in <em>A. annua</em> is confined to the glandular hairs on the plant, leading to low yields of artemisinin, our COSTREL tobacco lines produce it in their chloroplasts and therefore the whole leaf,&#x201D; says lead author and postdoctoral researcher Paulina Fuentes.&nbsp;</p>\n\n<p>&#x201C;We generated over 600 engineered tobacco plant lines that harbour different combinations of these additional genes, and analysed them in terms of the amounts of artemisinic compounds they acquired. We could then identify those that generated unprecedented levels of 120 milligrams per kilogram of artemisinic acid in their leaves, which can be readily converted into artemisinin through simple chemical reactions.&#x201D;&nbsp;</p>\n\n<p>Although further increases in these production levels will be needed if global demand for artemisinin is to be met, the study lays the foundation for much cheaper production of this life-saving therapy in a high-biomass crop, in contrast to a single medicinal plant.&nbsp;</p>\n\n<p>It also provides a new tool for engineering many other complex pathways, with the potential to increase production of other essential therapeutic ingredients.&nbsp;</p>\n\n<p>##&nbsp;</p>\n\n<p><strong>Reference</strong>&nbsp;</p>\n\n<p>The paper &#x2018;A new synthetic biology approach allows transfer of an entire metabolic pathway from a medicinal plant to a biomass crop&#x2019; can be freely accessed online at http://dx.doi.org/10.7554/eLife.13664. Contents, including text, figures, and data, are free to reuse under a CC BY 4.0 license.&nbsp;</p>\n\n<p><strong>Media contacts</strong>&nbsp;</p>\n\n<p>Emily Packer, eLife&nbsp;</p>\n\n<p>e.packer@elifesciences.org&nbsp;</p>\n\n<p>01223 855373&nbsp;</p>\n\n<p>Ulrike Glaubitz, Max Planck Institute of Molecular Plant Physiology&nbsp;</p>\n\n<p>glaubitz@mpimp-golm.mpg.de &nbsp;</p>\n\n<p>+49 331 567 8275&nbsp;</p>\n\n<p><strong>About eLife&nbsp;</strong></p>\n\n<p>eLife is a unique collaboration between the funders and practitioners of research to improve the way important research is selected, presented, and shared. eLife publishes outstanding works across the life sciences and biomedicine &#x2014; from basic biological research to applied, translational, and clinical studies. All papers are selected by active scientists in the research community. Decisions and responses are agreed by the reviewers and consolidated by the Reviewing Editor into a single, clear set of instructions for authors, removing the need for laborious cycles of revision and allowing authors to publish their findings quickly. eLife is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.&nbsp;</p>\n",
        [
          'content' => "<p>A new and inexpensive technique for mass-producing the main ingredient in the most effective treatment for malaria, artemisinin, could help meet global demands for the drug, according to a study to be published in the journal eLife. </p>\n\n<p>Artemisinin is produced in low yields by a herb called <em>Artemisia annua</em> (<em>A. annua</em>), otherwise known as sweet wormwood. Researchers from the Max Planck Institute of Molecular Plant Physiology have now discovered a new way to produce artemisinic acid, the molecule from which artemisinin is derived, in high yields. Their method involves transferring its metabolic pathway &#x2013; the series of biochemical steps involved in its production &#x2013; from <em>A. annua</em> into tobacco, a high-biomass crop. </p>\n\n<p>&#x201C;Malaria is a devastating tropical disease that kills almost half a million people every year,&#x201D; says contributing author Ralph Bock, Director of the Department for Organelle Biology, Biotechnology and Molecular Ecophysiology. </p>\n\n<p>&#x201C;For the foreseeable future, artemisinin will be the most powerful weapon in the battle against malaria but, due to its extraction from low-yielding plants, it is currently too expensive to be widely accessible to patients in poorer countries. Producing artemisinic acid in a crop such as tobacco, which yields large amounts of leafy biomass, could provide a sustainable and inexpensive source of the drug, making it more readily available for those who need it most.&#x201D; </p>\n\n<p>The team has called this approach to producing more artemisinic acid COSTREL (&#x201C;combinatorial supertransformation of transplastomic recipient lines&#x201D;). The first step in their process was to transfer the genes of the artemisinic acid pathway&#x2019;s core set of enzymes into the chloroplast genome of tobacco plants, generating what are known as transplastomic plants. </p>\n\n<p>The team then used their best transplastomic tobacco plant line to introduce an additional set of genes into its nuclear genome, generating the COSTREL lines. These remaining genes encode factors that increase the synthesis, or generation, of the acid in ways that are still largely unknown. </p>\n\n<p>&#x201C;While the artemisinic acid pathway in <em>A. annua</em> is confined to the glandular hairs on the plant, leading to low yields of artemisinin, our COSTREL tobacco lines produce it in their chloroplasts and therefore the whole leaf,&#x201D; says lead author and postdoctoral researcher Paulina Fuentes. </p>\n\n<p>&#x201C;We generated over 600 engineered tobacco plant lines that harbour different combinations of these additional genes, and analysed them in terms of the amounts of artemisinic compounds they acquired. We could then identify those that generated unprecedented levels of 120 milligrams per kilogram of artemisinic acid in their leaves, which can be readily converted into artemisinin through simple chemical reactions.&#x201D; </p>\n\n<p>Although further increases in these production levels will be needed if global demand for artemisinin is to be met, the study lays the foundation for much cheaper production of this life-saving therapy in a high-biomass crop, in contrast to a single medicinal plant. </p>\n\n<p>It also provides a new tool for engineering many other complex pathways, with the potential to increase production of other essential therapeutic ingredients. </p>\n\n<p>## </p>\n\n<p>",
        ],
      ],
      [
        "<p>Scientists have found a way of predicting the clinical progression of HIV in infected individuals by measuring the levels of HIV-1 (Human Immunodeficiency Virus Type 1) DNA in white blood cells. The technique might be used to calculate how long it takes for the virus to return once treatment has been suspended.</p>\n<div>\n  Results are published in the journal eLife by Dr John Frater and colleagues at the University of Oxford (UK), University College London (UK), Imperial College London (UK), and The Kirby Institute of New South Wales (Australia).&nbsp;</div>\n<div>\n  &nbsp;</div>\n<div>\n  HIV is currently an incurable, chronic infection that is manageable on antiretroviral treatment (ART). However, once a patient stops receiving this treatment the virus almost always returns. At present, the standard way of measuring the progression of the disease within a patient is by measuring the amount of HIV within a small sample of blood, a measure known as the viral load. High viral loads are predictive of a more rapid disease progression.</div>\n<div>\n  &nbsp;</div>\n<div>\n  The next step in advancing our ability to potentially cure HIV is to find a way to predict how long it will take for the virus to rebound in patients on ART treatment when therapy is interrupted. In their paper, the team suggests that by measuring HIV-1 DNA -- some of which has been integrated into the genetic material inside patients&rsquo; cells -- it is possible to gain a clearer picture of this timeline.</div>\n<div>\n  &nbsp;</div>\n<div>\n  To find the link between HIV-1 DNA levels and clinical progression of the disease, the group measured HIV-1 DNA levels in the white blood cells (CD4 T cells) of 154 HIV-positive individuals before anti-HIV therapy, at the point of stopping therapy, and at regular time intervals after therapy. They then compared these levels with known markers of clinical progression.</div>\n<div>\n  &nbsp;</div>\n<div>\n  They found that the level of HIV-1 DNA within a patient&rsquo;s white blood cells corresponded directly with the time taken for persistent, detectable levels of HIV in the blood to return, having been undetectable as a result of the therapy. The researchers hope to be able to use this finding to predict how long it will take the virus to rebound once a patient has stopped antiretroviral medication.</div>\n<div>\n  &nbsp;</div>\n<div>\n  &ldquo;Many researchers are striving to find a cure for HIV infection. Although still a distant goal, our data suggest that some people might be more curable than others, and that it might be possible to design a series of tests to use in the clinic to help identify them,&rdquo; said John Frater, MRC Senior Clinical Fellow at Oxford University and co-senior author. &ldquo;In this way we might be able to advise as to who should stay on their current treatment and who might be a candidate for either a trial of stopping therapy or even a new treatment in the future.&rdquo;</div>\n<div>\n  &nbsp;</div>\n<div>\n  ###</div>\n<div>\n  &nbsp;</div>\n<div>\n  <strong>Reference</strong></div>\n<div>\n  The paper &lsquo;HIV-1 DNA predicts disease progression and post-treatment virological control&rsquo; can be freely accessed online at <a href=\"http://elifesciences.org/lookup/doi/10.7554/elife.03821\">elifesciences.org/lookup/doi/10.7554/elife.03821</a>. Contents, including text, figures, and data, are free to re-use under a CC BY 4.0 license.</div>\n<div>\n  &nbsp;</div>\n<div>\n  <strong>Media contacts&nbsp;</strong></div>\n<div>\n  Jennifer Mitchell, eLife</div>\n<div>\n  <a href=\"mailto:j.mitchell@elifesciences.org\">j.mitchell@elifesciences.org</a></div>\n<div>\n  +44 (0) 1223 855 373</div>\n<div>\n  &nbsp;</div>\n<div>\n  &nbsp;</div>\n<div>\n  <strong>About eLife Sciences Publications Ltd</strong></div>\n<div>\n  &nbsp;</div>\n<div>\n  eLife is a unique collaboration between the funders and practitioners of research to communicate ground-breaking discoveries in the life and biomedical sciences in the most effective way. The eLife journal is a platform for maximising the reach and influence of new discoveries and showcasing new approaches to the presentation, use, and assessment of research. eLife is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.</div>\n",
        [
          'relatedContent' => [
            [
              'type' => 'article',
              'source' => '03821',
            ],
          ],
        ],
      ],
      [
        "<p><strong>Blood-sucking flies can act as 'flying syringes' to detect emerging infectious diseases in wild animals before they spread to humans, according to research published in the journal&nbsp;eLife.</strong></p>\n\n<p>The discovery has significant implications for our ability to control the global outbreak of new and re-emerging infections such as the Ebola and Zika viruses.</p>\n\n<p>Seven new pathogens are identified worldwide each year, and this is predicted to reach 15-20 every year by 2020 because of increased human contact with wildlife species that are potential reservoirs of disease. Despite significant scientific advances, researchers are still unable to predict where, when and how epidemics arise.</p>\n\n<p>\"This is a huge public health issue that urgently requires new tools for the active monitoring of outbreaks and rapid diagnosis of the pathogens involved,\" explains senior author and evolutionary geneticist Franck Prugnolle, from the National Center for Scientific Research (CNRS) in Montpellier, France. \"We wanted to investigate whether blood-feeding insects could act as a sampling tool out in the wild environment, allowing us to monitor the presence and emergence of infectious disease.\"</p>\n\n<p>Existing methods for studying the circulation of pathogens in wild habitats involve analysing animals caught for food, which represent only a fraction of a region's wildlife, or directly trapping animals to study the presence of infection in their organs and tissues, which is difficult and dangerous to protected species.</p>\n\n<p>Previous research had shown that DNA from host animals, and from pathogens such as malaria, is preserved in the blood meals of flies. This prompted Prugnolle and his team to see whether blood meals could be used as an indirect, non-invasive way of studying the circulation of pathogens in wild animals.</p>\n\n<p>Over a 16-week period, they conducted a field study in four national parks in the forests of Gabon, Central Africa, setting traps for three types of fly. They then analysed the insects' blood meals to determine the origin of the blood and the species of any malaria parasites present.</p>\n\n<p>More than 4,000 flies were captured, of which 30% - mostly tsetse flies, which spread African sleeping sickness - were engorged with blood. Lead author Paul-Yannick Bitome-Essono, from the National Center for Scientific and Technological Research, France, explains: \"We thought the tsetse fly might be a good candidate in our study, as both sexes feed on blood, they are large and easily trapped, present in large numbers in Central Africa, and are opportunistic feeders with no strong preference for a particular host animal, so would feed on a large range of wildlife.\"</p>\n\n<p>Using a new technique for closely studying host blood DNA, the team determined the host origin for three-quarters of these samples, showing that the flies had fed on over 20 different species ranging from elephants and hippopotamuses to reptiles and birds. They found malaria parasites in nearly 9% of the blood meals, including 18 cases of previously undocumented malaria species. The method also allowed them to identify the natural hosts of some malaria species whose preferred host was previously unknown.</p>\n\n<p>\"These results show that blood meals of the engorged flies can be successfully used to analyse the diversity of known malaria parasites,\" says Prugnolle.</p>\n\n<p>He adds that the next step is to look at ways to improve the method with next-generation sequencing and high-throughput pathogen detection methods. \"This approach of 'xenosurveillance' could detect pathogens before they spread to humans, as well as the emergence of new diseases in wild animals that may threaten their long-term survival.\"</p>\n\n<p>##</p>\n\n<p><strong>Reference</strong></p>\n\n<p>The paper 'Tracking zoonotic pathogens using blood-sucking flies as \"flying syringes\"' can be freely accessed online at&nbsp;<a href=\"http://dx.doi.org/10.7554/eLife.22069\" style=\"box-sizing: border-box; background: transparent; color: rgb(0, 136, 204); text-decoration: none; transition: all 0.1s linear;\" target=\"_blank\">http://dx.doi.org/10.7554/eLife.22069</a>. Contents, including text, figures, and data, are free to reuse under a CC BY 4.0 license.</p>\n\n<p><strong>Media contact</strong></p>\n\n<p>Emily Packer,&nbsp;eLife&nbsp;<br />\n<a href=\"mailto:e.packer@elifesciences.org\" style=\"box-sizing: border-box; background: transparent; color: rgb(0, 136, 204); text-decoration: none; transition: all 0.1s linear;\">e.packer@elifesciences.org</a>&nbsp;<br />\n01223 855373</p>\n\n<p>About&nbsp;eLife</p>\n\n<p>eLife&nbsp;is a unique collaboration between the funders and practitioners of research to improve the way important research is selected, presented and shared.&nbsp;eLife&nbsp;publishes outstanding works across the life sciences and biomedicine -- from basic biological research to applied, translational and clinical studies. All papers are selected by active scientists in the research community. Decisions and responses are agreed by the reviewers and consolidated by the Reviewing Editor into a single, clear set of instructions for authors, removing the need for laborious cycles of revision and allowing authors to publish their findings quickly.&nbsp;eLife&nbsp;is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.</p>\n",
        [
          'mediaContacts' => NULL,
        ],
      ],
      [
        "<p><strong>In the first study to use new methods&nbsp;to&nbsp;track antibiotic resistance through the process of intensively farming and slaughtering cattle, scientists have discovered a “startling” lack of resistance genes in meat.&nbsp;</strong></p>\n\n<p>Meanwhile, in soil and faeces samples from cattle pens they found genes resistant to a powerful “last resort” class of antibiotics called carpabemens that aren’t used in the livestock industry. These genes may have jumped from humans or companion animals to livestock, or could even be present at low levels in the wider environment.</p>\n\n<p>Together, the results published in <a href=\"http://dx.doi.org/10.7554/eLife.13195\">eLife</a> suggest researchers and policy-makers need to switch focus to combat the growing problem of drug-resistant bugs. A current focus for policy-makers is to reduce antibiotic use in livestock to curb the spread of drug-resistant bugs. The team urges that traffic from humans to animals, and back to humans via the environment, should be a new focus for research.</p>\n\n<p>“Our findings clearly show that the spread of resistance is not a one-way street from animals to humans and that, as new evidence emerges, we need to shift focus ,” says lead author Noelle Noyes from the Microbial Ecology Group at Colorado State University.</p>\n\n<p>The lack of resistance genes in post-slaughter meat samples was a big surprise for the scientists, forcing them to rethink the view that it is only antibiotic use that increases resistance.</p>\n\n<p>“While we expected to find fewer bacteria and thus resistance genes, the absence of resistance genes in these samples was still a bit startling,” says co-principal investigator Paul Morley from Colorado State University.</p>\n\n<p>Strict, technology-driven food safety measures prevent pathogenic bacteria like <em>Salmonella</em> and <em>E. coli</em> O157:H7 entering the food supply chain. They include high heat, steam, organic acids and cutting off parts of the carcass at risk of harbouring pathogens.</p>\n\n<p>“Our findings suggest the gauntlet of measures to kill pathogens also protects the consumer from antimicrobial resistance genes because they too are unable to survive,” says co principal investigator Keith Belk.</p>\n\n<p>“We need to expand our thinking in this area, and develop new and improved methods to better understand how antibiotic use drives a complex network of genetic modifications within entire microbial communities,” he says.</p>\n\n<p>Environmental routes of exposure are much harder to trace and have been largely overlooked by researchers and policy-makers. While many of us never step foot on working farms, we are physically connected to agriculture via wastewater run-off and windborne particulates. The scientists suggest investigating wind patterns and water flow to see if, and how, resistant bacteria may be disseminated, and how far.&nbsp;</p>\n\n<p>“We may find that such dissemination is very limited geographically, or we may find that resistant bacteria can travel long distances if they find the right currents or the right waterways. In either case, this would be very important information from a public health perspective,” says Noyes.</p>\n\n<p>The researchers collected samples from 1,741 commercial cattle. The study started in feedlots, where intensively farmed cattle are moved after grazing. A feedlot consists of outdoor pens where cattle are fattened during their final months of life.&nbsp; Samples were also taken during slaughter and from market-ready products. No previous studies have tracked antimicrobial use and resistance right through the beef production process. The team found no resistance genes to any bacteria in market-ready beef products.</p>\n\n<p>They did discover changes to antibiotic resistance genes in the guts of cattle during their time in the feedlot. The changes could be due to the use of antibiotics in feedlots but could also result from adjusting to a high-energy diet or from the cattle’s maturation from adolescent to adult.</p>\n\n<p>The diversity of genes in their ‘resistome’ decreased. A resistome is the collection of antibiotic resistance genes in a given environment, be it the gut of a cow or a sample of soil or water. The decrease could present an opportunity to suppress it further during the feeding period and move towards the lowest risk possible.</p>\n\n<p>“The next challenge is to identify what is really driving the change we saw and to determine whether these drivers need to be modified and, if so, how,” says Noyes.</p>\n\n<p>“Any changes need to be balanced with the ability of agriculture to produce enough safe, affordable food for a rapidly changing population. What concerns us most is ensuring that rational, science-driven discussion drives progress,” she says.</p>\n\n<p>##</p>\n\n<p><strong>Reference</strong></p>\n\n<p>The paper ‘Resistome diversity in cattle and the environment decreases during beef production’ can be freely accessed online at <a href=\"http://dx.doi.org/10.7554/eLife.13195\">http://dx.doi.org/10.7554/eLife.13195</a>. Contents, including text, figures, and data, are free to reuse under a CC BY 4.0 license.</p>\n\n<p><strong>Media contacts</strong></p>\n\n<p>Zoe Dunford</p>\n\n<p><a href=\"mailto:z.dunford@elifesciences.org\">z.dunford@elifesciences.org</a></p>\n\n<p>+44 (0)7786 303597</p>\n\n<p>Mary Guiden</p>\n\n<p><a href=\"mailto:Mary.guiden@colostate.edu\">Mary.guiden@colostate.edu</a></p>\n\n<p>(970) 491-6892</p>\n\n<p><strong>About eLife</strong></p>\n\n<p>eLife is a unique collaboration between the funders and practitioners of research to improve the way important research is selected, presented, and shared. eLife publishes outstanding works across the life sciences and biomedicine — from basic biological research to applied, translational, and clinical studies. All papers are selected by active scientists in the research community. Decisions and responses are agreed by the reviewers and consolidated by the Reviewing Editor into a single, clear set of instructions for authors, removing the need for laborious cycles of revision and allowing authors to publish their findings quickly. eLife is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.</p>\n",
        [
          'mediaContacts' => [
            [
              'name' => [
                'preferred' => 'Mary Guiden',
                'index' => 'Guiden, Mary',
              ],
              'emailAddresses' => [
                'Mary.guiden@colostate.edu',
              ],
              'phoneNumbers' => [
                '+19704916892',
              ],
            ],
          ],
        ],
      ],
      [
        "<div>\n<p><strong>The first results from the Reproducibility Project: Cancer Biology have been published in eLife. The aim of the project, which is a collaboration between the Center for Open Science and Science Exchange, is to assess reproducibility in cancer biology, and to identify what influences its success or failure in science more generally.</strong></p>\n\n<p>An eLife editorial published to coincide states: &#x201C;Reproducibility is a cornerstone of science, and the development of new drugs and medical treatments relies on the results of preclinical research being reproducible. In recent years, however, the validity of published findings in a number of areas of scientific research, including cancer research, have been called into question (Begley and Ellis, 2012; Baker, 2016)&#x201D;.&nbsp;</p>\n\n<p>At the same time, replicating other researchers&#x2019; work can be onerous and there is little incentive for researchers to do so. Many journals prefer to publish new results.</p>\n\n<p>&#x201C;The first five papers are part of a substantial effort to study reproducibility in cancer biology in a transparent way,&#x201D; says eLife Editor-in-Chief Randy Schekman. &#x201C;We aim to bolster this effort by applying eLife&#x2019;s rigorous and consultative editorial process to the evaluation and peer review of the resulting papers.&#x201D;</p>\n\n<p>&#x201C;This is an experiment. Every approach to assessing reproducibility has strengths and weaknesses. With over 20 further papers to publish, we do not yet know whether the approach taken in this project is the most effective. However, these initial papers will encourage discussion of these issues and the final analyses will highlight factors that influence reproducibility and the best ways to evaluate it,&#x201D; he says.</p>\n\n<p>Tim Errington from the Center for Open Science says: &#x201C;These first five papers show that achieving reproducibility is hard and there is room for improvement. We can already see how variations in the availability of critical information and processes can impact how challenging it is to replicate projects and results.&#x201D;</p>\n\n<p>The replication studies in the cancer biology project are presented in two distinct phases. First, the project leaders developed protocols for each study and shared those with original authors for informal review, before submitting the protocols as &#x201C;Registered Reports&#x201D; to eLife for formal peer review.&nbsp;</p>\n\n<p>Registered Reports are an opportunity to get expert feedback from reviewers to maximize the quality of the experimental design and methodology.&nbsp;</p>\n\n<p>Next, Replication Studies are conducted as specified in the Registered Reports and are also subject to eLife peer review.</p>\n\n<p>&#x201C;We are proud to facilitate this landmark project. Science Exchange provides efficient access via our unique marketplace to the world-class scientific expertise required to independently conduct these studies,&#x201D; says founder and CEO of Science Exchange Elizabeth Iorns.</p>\n\n<p>The assessment of eLife editors is that of the first five Replication Studies, two broadly supported key conclusions from the original studies, two were inconclusive due to technical problems with certain key experiments, and one failed to reproduce the findings of the original study.</p>\n\n<p>The editorial concludes: &#x201C;We will publish more Replication Studies over the months ahead and, at the conclusion of the project, a meta-analysis of all the studies. While we wait for this, it is important not to overinterpret the results. Already it is clear that nuanced interpretations are necessary, not black and white conclusions about which studies reproduced and which did not. It is also clear that this approach to testing reproducibility remains an experiment, with advantages and disadvantages, including the fact that it sometimes yields results that cannot be interpreted.&#x201D;</p>\n\n<p>The original studies were published between 2010 and 2012. By the end of the project, around 30 Replication Studies will have been reviewed by eLife.</p>\n\n<p>##</p>\n\n<p><strong>References</strong></p>\n\n<p>The five Replication Studies, along with an Editorial and other supporting articles, can be freely accessed online. Contents, including text, figures, and data, are free to reuse under a CC BY 4.0 license.</p>\n\n<p>Please link or hyperlink to papers via the DOI:</p>\n\n<p><strong>The Replication Studies:</strong></p>\n\n<p>http://dx.doi.org/10.7554/eLife.17044<br />\nhttp://dx.doi.org/10.7554/eLife.17584<br />\nhttp://dx.doi.org/10.7554/eLife.18173<br />\nhttp://dx.doi.org/10.7554/eLife.21253<br />\nhttp://dx.doi.org/10.7554/eLife.21634</p>\n\n<p><strong>Editorial:</strong></p>\n\n<p>http://dx.doi.org/10.7554/eLife.23693&nbsp;</p>\n\n<p><strong>Other Content:</strong></p>\n\n<p>Feature Article: Making sense of replications: http://dx.doi.org/10.7554/eLife.23383<br />\nInsight article on Replication Study 17044: http://dx.doi.org/10.7554/eLife.22661<br />\nInsight article on Replication Study 21253: http://dx.doi.org/10.7554/eLife.22915<br />\nInsight article on Replication Study 21634: http://dx.doi.org/10.7554/eLife.22662</p>\n\n<p><strong>Media contacts</strong></p>\n\n<p>Zoe Dunford, eLife: z.dunford@elifesciences.org, +44 (0)7786 303597<br />\nCenter for Open Science Research: Tim Errington: tim@cos.io<br />\nCenter for Open Science Media: Rusty Speidel: rusty@cos.io<br />\nScience Exchange: Elizabeth Iorns: Elizabeth@scienceexchange.com<br />\nScience Exchange: Nicole Perfito: Nicole@scienceexchange.com<br />\nScience Exchange: Keith Osiewicz: Keith@scienceexchange.com&nbsp;</p>\n\n<p><strong>About eLife</strong></p>\n\n<p>eLife is a unique collaboration between the funders and practitioners of research to improve the way important research is selected, presented, and shared. eLife publishes outstanding works across the life sciences and biomedicine &#x2014; from basic biological research to applied, translational, and clinical studies. All papers are selected by active scientists in the research community. Decisions and responses are agreed by the reviewers and consolidated by the Reviewing Editor into a single, clear set of instructions for authors, removing the need for laborious cycles of revision and allowing authors to publish their findings quickly. eLife is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.</p>\n\n<p><strong>About the Center for Open Science</strong></p>\n\n<p>The Center for Open Science (COS) is a non-profit technology startup founded in 2013 with a mission to increase openness, integrity, and reproducibility of scientific research. COS pursues this mission by building communities around open science practices, supporting metascience research, and developing and maintaining free, open source software tools. The Open Science Framework (OSF), COS&#x2019;s flagship product, is a web application that connects and supports the research workflow, enabling scientists to increase the efficiency and effectiveness of their research. Researchers use the OSF to collaborate, document, archive, share, and register research projects, materials, and data. Learn more at cos.io and osf.io.</p>\n\n<p><strong>About Science Exchange</strong></p>\n\n<p>Science Exchange is the world&#x2019;s leading marketplace for scientific research services. The company provides secure access to a network of over 3000 screened and verified contract research organizations (CROs), academic labs, and government facilities that are available to conduct experiments on the behalf of scientists. The Science Exchange platform has been used by scientists from over 2,500 different companies and organizations, solving one of the most significant challenges facing the highly-trained researchers at these companies: time and resources spent identifying and managing outsourced research projects. To date, Science Exchange has raised over $30 million from Maverick Capital Ventures, Union Square Ventures, Index Ventures, OATV, the YC Continuity Fund, and others. For more information visit www.scienceexchange.com.</p>\n</div>\n",
        [
          'mediaContacts' => NULL,
        ],
      ],
      [
        "<div>\n<p><strong>A type of flatworm could be a new weapon in the hunt for better ways to treat a group of diseases that can cause extreme sensitivity to light, facial hair growth, and hallucinations, according to a study published in the journal eLife.</strong></p>\n\n<p>Porphyrias are a group of rare metabolic disorders characterized by red and purple pigments accumulating in the body. With the accidental discovery that the skin color of the flatworm <em>Schmidtea mediterranea</em> (<em>S. mediterranea</em>) changes under prolonged exposure to sunlight, these animals could provide a new model for studying the diseases.</p>\n\n<p>Faculty and student researchers from Keene State College, New Hampshire, show that the flatworm generates light-activated molecules called porphyrins in its skin pigment cells using the same biochemical pathway as that involved in human porphyrias.</p>\n\n<p>Their research developed from the unexpected observation by undergraduate students that the flatworms they were investigating changed color from brown to white when exposed to sunlight for several days. It could now lead to future drug development to meet patient demands for more effective treatments.</p>\n\n<p>&#x201C;Porphyrias are typically caused by inherited mutations that involve a buildup of porphyrins in various parts of the body. Although naturally created during the generation of heme, a substance required for oxygen transport in red blood cells, porphyrins can have toxic effects when their levels increase,&#x201D; says Jason Pellettieri, corresponding author and Associate Professor of Biology at Keene State College.</p>\n\n<p>&#x201C;For example, porphyrins deposited in the skin cause swelling, blistering, and lesions upon exposure to bright light. Neurological issues can also arise, ranging from anxiety and confusion to seizures or paralysis. These episodes, which can last for weeks, can be triggered by drugs, hormonal changes, and dieting or fasting.&#x201D;</p>\n\n<p>The porphyrins in <em>S. mediterranea</em> give rise to their normal skin pigmentation. When the animals are exposed to intense light for extended periods of time (a situation unlikely to occur in the wild), porphyrin production leads to pigment cell loss, changing the animals&#x2019; skin color to white.</p>\n\n<p>To investigate this photosensitivity, the researchers first tested infrared and ultraviolet lights on the flatworms, which had no effect on their skin. In contrast, intense visible light altered their coloring, with just over half of them developing one or more small tissue lesions.</p>\n\n<p>&#x201C;Our findings show that prolonged light exposure eliminates the flatworms&#x2019; pigment cells through a mechanism involving porphyrin-dependent photosensitization. The animals then repigment when they are no longer exposed to light,&#x201D; explains lead author Bradford Stubenhaus.</p>\n\n<p>During the course of the research, the team also noticed a positive relationship between how long the flatworms were fasted before light exposure and the extent of their photosensitivity.</p>\n\n<p>To document this relationship, they starved the animals for one, seven, 14 or 30 days before exposing them to light. Depigmentation was strongly accelerated with starvation, and then reversed with a single feeding 24 hours before light exposure.</p>\n\n<p>These discoveries suggest that <em>S. mediterranea</em> could help identify new treatments for easing porphyrin-mediated photosensitivity or separating the relationship between dieting or fasting and the onset of disease symptoms.</p>\n\n<p>&#x201C;Although porphyrias are usually manageable diseases, reliance on the mainstay treatment, namely intravenous heme, or liver transplantation, for more severe cases can result in significant complications. There are also no approved preventative therapies for patients who suffer recurrent attacks,&#x201D; says Pellettieri.</p>\n\n<p>&#x201C;Our studies show that flatworms such as <em>S. mediterranea</em> could potentially change this. They have recently emerged as a useful model for human disorders, including Usher syndrome &#x2013; a genetic disorder that affects vision, hearing, and sometimes balance &#x2013; and cystic kidney disease. We can now add porphyria to this growing list, with plans to use the animals to screen for novel therapies in the near future.&#x201D;</p>\n\n<p>##</p>\n\n<p><strong>Reference</strong></p>\n\n<p>The paper &#x2018;Light-induced depigmentation in planarians models the pathophysiology of acute porphyrias&#x2019; can be freely accessed online at <a href=\"http://dx.doi.org/10.7554/eLife.14175\">http://dx.doi.org/10.7554/eLife.14175</a>. Contents, including text, figures, and data, are free to reuse under a CC BY 4.0 license.</p>\n\n<p><strong>Media contacts</strong></p>\n\n<p>Emily Packer, eLife</p>\n\n<p><a href=\"mailto:e.packer@elifesciences.org\">e.packer@elifesciences.org</a></p>\n\n<p>01223 855373</p>\n\n<p>Kelly Ricaurte, Keene State College</p>\n\n<p><a href=\"mailto:kricaurte@keene.edu\">kricaurte@keene.edu</a></p>\n\n<p>603-358-2119</p>\n\n<p><strong>About eLife</strong></p>\n\n<p>eLife is a unique collaboration between the funders and practitioners of research to improve the way important research is selected, presented, and shared. eLife publishes outstanding works across the life sciences and biomedicine &#x2014; from basic biological research to applied, translational, and clinical studies. All papers are selected by active scientists in the research community. Decisions and responses are agreed by the reviewers and consolidated by the Reviewing Editor into a single, clear set of instructions for authors, removing the need for laborious cycles of revision and allowing authors to publish their findings quickly. eLife is supported by the Howard Hughes Medical Institute, the Max Planck Society, and the Wellcome Trust. Learn more at elifesciences.org.</p>\n\n<p><strong>About Keene State College</strong></p>\n\n<p>Keene State College is a preeminent public liberal arts college that ensures student access to world-class academic programs. Integrating academics with real-world application and active community and civic engagement, Keene State College prepares graduates to meet society&#x2019;s challenges by thinking critically, acting creatively, and serving the greater good. To learn more about Keene State College, visit www.keene.edu.</p>\n</div>\n",
        [
          'mediaContacts' => [
            [
              'name' => [
                'preferred' => 'Kelly Ricaurte',
                'index' => 'Ricaurte, Kelly',
              ],
              'emailAddresses' => [
                'kricaurte@keene.edu',
              ],
              'phoneNumbers' => [
                '+16033582119',
              ],
              'affiliations' => [
                [
                  'name' => [
                    'Keene State College',
                  ],
                ],
              ],
            ],
          ],
        ],
      ],
    ];
  }

}
